poofarmer/project.godot

```diff
@@ -29,7 +29,7 @@ _global_script_classes=[ {
 "language": "GDScript",
 "path": "res://Scripts/Gnome.gd"
 }, {
-"base": "Enemy",
```

Comment:
This should probably stay an `Enemy`

Reply:
I don't remember changing this. I'll take a look at it.

Reply:
Okay this should be fixed now